### ISSUE TITLE
Update docker readme

### ISSUE THREAD

**Issue by Jyyjy** [apache/flagon#47]
https://github.com/apache/flagon/blob/master/docker/README.md

**Comment by krishna-dave206** [comment]
@Jyyjy I can work on this issue. Can you brief me more about the issue?

**Comment by Jyyjy** [comment]
Yes, let me update the description. I think I created this issue for myself but never ended up completing it.

**Comment by krishna-dave206** [comment]
@Jyyjy I will complete this but it would be better for me if you can provide specific things to add in the readme file.

**Comment by krishna-dave206** [comment]
@Jyyjy I have raised the PR. Pls review it and if everything is ok, kindly merge it.
